src/playlist/playlistmanager.cpp

```diff
@@ -494,6 +494,11 @@ void PlaylistManager::RemoveItemsWithoutUndo(int id,
   playlists_[id].p->RemoveItemsWithoutUndo(indices);
 }
 
+void PlaylistManager::RemoveCurrentSong()
+{
```

Comment:
`{` on the same line as the function like everywhere else or just run `make format`.